components/workflow-mgt/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/dao/BPSProfileDAO.java

```diff
@@ -51,7 +49,7 @@ public void addProfile(BPSProfile bpsProfileDTO, int tenantId)
         Connection connection = IdentityDatabaseUtil.getDBConnection();
         PreparedStatement prepStmt = null;
         String query = SQLConstants.ADD_BPS_PROFILE_QUERY;
-        String password = bpsProfileDTO.getPassword();
+        String password = String.copyValueOf(bpsProfileDTO.getPassword());
```

Comment:
We encrypt this password and store encrypted password as String to database, should I store encrypted password as char[]?